src/app/modal/modal.controller.js

```diff
@@ -151,36 +140,102 @@ class CareerPortalModalController {
-                } else {
-                    if (userProfile.positions.values[0].isCurrent) {
-                        this.linkedInData.resume += this.$filter('i18n')('modal.present');
+            var positions = userProfile.positions.values;
+            // Iterate through each position
+            if (positions && positions.length) {
+                for (var ii = 0; ii < positions.length; ii++) {
+                    // Add Employee section header
+                    this.linkedInData.resume += (positions[ii].company.name || '') + ' ';
+                    // Start Date
+                    if (positions[ii].startDate) {
+                        this.linkedInData.resume += months[positions[ii].startDate.month - 1] + ' ' + positions[ii].startDate.year + ' - ' || '';
+                    }
+                    // End Date or 'Present'
+                    if (positions[ii].endDate) {
+                        this.linkedInData.resume += months[positions[ii].endDate.month - 1] + ' ' + positions[ii].endDate.year || '';
+                    } else {
+                        if (positions[ii].isCurrent) { // jshint ignore:line
+                            this.linkedInData.resume += this.$filter('i18n')('modal.present');
+                        }
+                    }
+                    this.linkedInData.resume += lineBreak;
+                    // Title
+                    this.linkedInData.resume += positions[ii].title + lineBreak || '';
+                    // Industry
+                    this.linkedInData.resume += positions[ii].company.industry ? positions[ii].company.industry + lineBreak : '';
+                    if (positions[ii].location && positions[ii].location.name) {
+                        // Locale
+                        this.linkedInData.resume += positions[ii].location.name + lineBreak || '';
+                    }
+                    // Summary
+                    if (positions[ii].summary) {
+                        this.linkedInData.resume += positions[ii].summary + lineBreak || '';
                     }
+                    this.linkedInData.resume += hardBreak;
                 }
-                this.linkedInData.resume += lineBreak;
-                // Title
-                this.linkedInData.resume += userProfile.positions.values[0].title + lineBreak || '';
-                // Industry
-                this.linkedInData.resume += userProfile.positions.values[0].company.industry ? userProfile.positions.values[0].company.industry + lineBreak : '';
-                if (userProfile.positions.values[0].location) {
-                    // Locale
-                    this.linkedInData.resume += userProfile.positions.values[0].location.name + lineBreak || '';
+            }
+        }
+
+        // Skills
+        if (userProfile.skills && userProfile.skills.values) {
```

Comment:
Consider following same value-check and iteration code as decided for the above.